packages/component/src/Activity/StackedLayout.js

```diff
@@ -162,6 +162,7 @@ const StackedLayout = ({
               // Disable "Prop `id` is forbidden on DOM Nodes" rule because we are using the ID prop for accessibility.
               /* eslint-disable-next-line react/forbid-dom-props */
               id={ariaLabelId}
+              role="log"
```

Comment:
First, `role="log"` implies `aria-live="polite"`. We shouldn't use this role.

Second, this is the message bubble, not a transcript.

Reply:
Then what do you suggest?

Reply:
https://www.w3.org/WAI/PF/aria/roles

Let's pick one of the role here.